docs/quick-start.md

```diff
@@ -94,7 +94,7 @@ kubectl delete -f https://raw.githubusercontent.com/numaproj/numaflow/stable/exa
 
 Now we will walk you through creating an advanced pipeline. In our example, this is called the `even-odd` pipeline, illustrated by the following diagram:
 
-![Pipeline Diagram](assets/even-odd.png)
+![Pipeline Diagram](assets/even-odd-square.png)
```

Comment:
Name does not match, PNG vs. png.

If we use the new image, remove the old one from assets.

Reply:
should be good now. thanks!